extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcTestConfig.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.arc.deployment;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class ArcTestConfig {
+
+    /**
+     * If set to true then disable {@code StartupEvent} observers declared on application bean classes during the tests.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean disableApplicationStartupObservers;
```

Comment:
I am wondering if it makes sense to disable the `ShutdownEvent` as well

Reply:
That's a good point. In general, the `ShutdownEvent` is less often used because one can simply use `@PreDestroy` on application/singleton beans but it probably makes sense to keep it consistent.

Reply:
Yeah, agreed.

Reply:
Done.